checkov/sca_package/output.py

```diff
@@ -82,9 +82,12 @@ def create_report_record(
     fixed_versions: List[Union[packaging_version.Version, packaging_version.LegacyVersion]] = []
     status = vulnerability_details.get("status") or "open"
     if status != "open":
-        fixed_versions = [
-            packaging_version.parse(version.strip()) for version in status.replace("fixed in", "").split(",")
-        ]
+        parsed_current_version = packaging_version.parse(package_version)
+        for version in status.replace("fixed in", "").split(","):
+            parsed_version = packaging_version.parse(version.strip())
+            if parsed_version > parsed_current_version:
```

Comment:
What kind of comparison is it? I mean, string comparison?

Reply:
nope, the module `packaging_version` creates instances of the class `Version` and it supports comparison